### PR TITLE
16 use expressrouter to manage routes

### LINKED ISSUE #16: Use expressRouter to manage routes
Manage routes using Router from Express

### PR DESCRIPTION
**Pull request by Meliodas-1O**
Closes #16

### CODE REVIEW

**Comment by maximekuhn** [comment]
Please merge main into this branch first

File: src/database/.env
Comment (maximekuhn) [inline src/database/.env:1]: why move here ?

File: test/integration_tests/chapters/chapter.integration.test.ts [hunk 1]
```diff
@@ -73,6 +73,8 @@ describe("Chapter integration tests suite", () => {
 
           test("5- Update chapter", async () => {
                // Given
+               console.log(id);
+
```
Comment (maximekuhn) [inline test/integration_tests/chapters/chapter.integration.test.ts:77]: we should remove this

File: test/integration_tests/chapters/chapter.integration.test.ts [hunk 2]
```diff
@@ -85,6 +87,8 @@ describe("Chapter integration tests suite", () => {
                     .set("Content-Type", "application/json");
 
                // Then
+               console.log(response.body);
+
```
Comment (maximekuhn) [inline test/integration_tests/chapters/chapter.integration.test.ts:91]: also remove this

File: test/integration_tests/exercises/exercise.integration.test.ts [hunk 3]
```diff
@@ -87,6 +87,7 @@ describe("Exercice integration tests suite", () => {
                     .set("Content-Type", "application/json");
 
                // Then
+               console.log(response.body);
```
Comment (maximekuhn) [inline test/integration_tests/exercises/exercise.integration.test.ts:90]: and remove this too

**Review by maximekuhn — CHANGES_REQUESTED** [review]
GG it's much cleaner now !
Needs some small fixes and then let's merge !

File: test/integration_tests/chapters/chapter.integration.test.ts (hunk 1 above)
Comment (Meliodas-1O) [reply test/integration_tests/chapters/chapter.integration.test.ts]: done

File: test/integration_tests/chapters/chapter.integration.test.ts (hunk 2 above)
Comment (Meliodas-1O) [reply test/integration_tests/chapters/chapter.integration.test.ts]: done

File: test/integration_tests/exercises/exercise.integration.test.ts (hunk 3 above)
Comment (Meliodas-1O) [reply test/integration_tests/exercises/exercise.integration.test.ts]: done

**Review by maximekuhn — APPROVED** [review]
GG my guy you can merge !

**Pull request merged**